src/main/java/cn/nukkit/inventory/PlayerInventory.java

```diff
@@ -235,6 +237,11 @@ public boolean setItem(int index, Item item) {
         return setItem(index, item, true, false);
     }
 
+    @Override
+    public boolean setItem(int index, Item item, boolean send) {
+        return this.setItem(index, item, send, false);
+    }
+
```

Comment:
Could you explain the need of this method override? As this method is not used in any diffs with regards to this PR.